.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Closes
+
+# Title
+
+## Contributing Guidelines
+
+If you haven't, go over the Contributing Guidelines.
+
+> Put an `x` in the box if you did.
+
+- [ ] I have read the **CONTRIBUTING** document.
+
+## Description
+
+> Describe your changes in detail.
```

Comment:
It's possible that this may only refer to the original issue if it already has enough context. If anything changed since the original issue was logged, it's important to make a note of it here.